src/main/java/ro/societateahermes/backendservice/service/serviceImplementation/EventServiceImplementation.java

```diff
@@ -0,0 +1,30 @@
+package ro.societateahermes.backendservice.service.serviceImplementation;
+
+import org.springframework.stereotype.Service;
+import ro.societateahermes.backendservice.entities.Event;
+import ro.societateahermes.backendservice.entities.Participation;
+import ro.societateahermes.backendservice.repository.EventRepositoryInterface;
+import ro.societateahermes.backendservice.service.EventServiceInterface;
+
+import java.util.List;
+
+
+@Service
+public class EventServiceImplementation implements EventServiceInterface {
+
+    private final EventRepositoryInterface eventRepository;
+
+    public EventServiceImplementation(EventRepositoryInterface eventRepository) {
+        this.eventRepository = eventRepository;
+    }
+
+
+    @Override
+    public void addParticipation(long eventID, Participation participation) {
```

Comment:
La fel ca mai sus. Ar trebui sa primesti de la controller un ParticipationDto, iar service-ul sa foloseasca un mapper ca sa converteasca din dto in entitate si invers